src/Symfony/Component/Notifier/Bridge/Smsmode/Tests/SmsmodeTransportTest.php

```diff
@@ -81,6 +83,44 @@ public function testNoInvalidArgumentExceptionIsThrownIfFromIsValid(string $from
         self::assertSame('foo', $sentMessage->getMessageId());
     }
 
+    public function testIfMandatoryHttpClientRequestHeaderContentTypeIsMissing()
```

Comment:
This is not testing anything here as you are not covering any code of the transport (you are not even using the transport)